content/docs/deployment/csmoperator/_index.md

```diff
@@ -0,0 +1,253 @@
+To uninstall a CSM operator, run `bash scripts/uninstall.sh`. This will uninstall the operator in `dell-csm-operator` namespace.
+
+{{< imgproc uninstall.jpg Resize "2500x" >}}{{< /imgproc >}}
+
+#### Operator Uninstallation on an environment with OLM
+To uninstall a CSM operator installed with OLM run `bash scripts/uninstall_olm.sh`. This will uninstall the operator in  `test-csm-operator-olm` namespace.
+
+{{< imgproc uninstall_olm.jpg Resize "2500x" >}}{{< /imgproc >}}
+
+## Custom Resource Definitions
+As part of the Dell CSM Operator installation, a CRD representing configuration for the CSI Driver and CSM Modules is also installed.  
+CRD which is installed in API Group `storage.dell.com`
+* containerstoragemodule
+
+For installation of the supported drivers and modules, a `CustomResource` has to be created in your cluster.
+
+## Supported CSI Drivers
+
+| CSI Driver         | Version   | ConfigVersion  |
+| ------------------ | --------- | -------------- |
+| CSI PowerScale     | 2.2.0     | v2.2.0         |
+
+## Pre-Requisites for installation of the CSI Drivers
+
+On upstream Kubernetes clusters, make sure to install
+* VolumeSnapshot CRDs - Install v1 VolumeSnapshot CRDs
+* External Volume Snapshot Controller with the correct version
+
+#### Volume Snapshot CRD's
+The Kubernetes Volume Snapshot CRDs can be obtained and installed from the external-snapshotter project on Github. Manifests are available here:[v5.0.x](https://github.com/kubernetes-csi/external-snapshotter/tree/v5.0.1/client/config/crd)
+
+#### Volume Snapshot Controller
+The CSI external-snapshotter sidecar is split into two controllers:
+- A common snapshot controller
+- A CSI external-snapshotter sidecar
+
+The common snapshot controller must be installed only once in the cluster irrespective of the number of CSI drivers installed in the cluster. On OpenShift clusters, the common snapshot-controller is pre-installed. In the clusters where it is not present, it can be installed using `kubectl` and the manifests are available here: [v5.0.x](https://github.com/kubernetes-csi/external-snapshotter/tree/v5.0.1/deploy/kubernetes/snapshot-controller)
+
+*NOTE:*
+- The manifests available on GitHub install the snapshotter image:
+    - [quay.io/k8scsi/csi-snapshotter:v5.0.1](https://quay.io/repository/k8scsi/csi-snapshotter?tag=v5.0.1&tab=tags)
+- The CSI external-snapshotter sidecar is still installed along with the driver and does not involve any extra configuration.
+
+#### Installation example
+
+You can install CRDs and the default snapshot controller by running the following commands:
+```bash
+git clone https://github.com/kubernetes-csi/external-snapshotter/
+cd ./external-snapshotter
+git checkout release-<your-version>
+kubectl create -f client/config/crd
+kubectl create -f deploy/kubernetes/snapshot-controller
+```
+
+*NOTE:*
+- It is recommended to use 5.0.x version of snapshotter/snapshot-controller.
+
+
+#### iSCSI
```

Comment:
ISCSI is not applicable to PowerScale Driver, need to remove this section for now.

Reply:
Removed this section since it's not applicable